op-service/backoff/strategies.go

```diff
@@ -16,24 +16,23 @@ type Strategy interface {
 // ExponentialStrategy performs exponential backoff. The exponential backoff
 // function is min(e.Min + (2^attempt * 1000) + randBetween(0, e.MaxJitter), e.Max)
 type ExponentialStrategy struct {
-	// Min is the minimum amount of time to wait between attempts in ms.
-	Min float64
+	// Min is the minimum amount of time to wait between attempts.
+	Min time.Duration
 
-	// Max is the maximum amount of time to wait between attempts in ms.
-	Max float64
+	// Max is the maximum amount of time to wait between attempts.
+	Max time.Duration
 
-	// MaxJitter is the maximum amount of random jitter to insert between
-	// attempts in ms.
-	MaxJitter int
+	// MaxJitter is the maximum amount of random jitter to insert between attempts.
+	MaxJitter time.Duration
 }
 
 func (e *ExponentialStrategy) Duration(attempt int) time.Duration {
-	var jitter int
+	var jitter time.Duration
 	if e.MaxJitter > 0 {
-		jitter = rand.Intn(e.MaxJitter)
+		jitter = time.Duration(rand.Int63n(e.MaxJitter.Milliseconds()))
```

Comment:
`time.Duration` is in nanoseconds, when creating one, you always need to multiple by the unit.

Reply:
Yes, thanks! didnt know it when making the changes